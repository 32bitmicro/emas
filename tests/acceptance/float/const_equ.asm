.const	f1 1.2345678
.equ	f2 -345768345768.123
.float	f1, f2
