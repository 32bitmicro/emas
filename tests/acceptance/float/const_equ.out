@ 0x0000 : 0x4f03  /  010 011 1 100 000 011  /  20227
@ 0x0001 : 0x28a9  /  001 010 0 010 101 001  /  10409
@ 0x0002 : 0x6c01  /  011 011 0 000 000 001  /  27649
@ 0x0003 : 0xaf7e  /  101 011 1 101 111 110  /  44926
@ 0x0004 : 0x99bf  /  100 110 0 110 111 111  /  39359
@ 0x0005 : 0x5727  /  010 101 1 100 100 111  /  22311
